frontend/src/pages/Team/Team.module.scss

```
@@ -4,35 +4,84 @@
   gap: var(--s3);
 }
 
-.summary {
+/* 팀 목표 카드. 대시보드 매출 목표 타일과 같은 어휘를 한 단계 크게 씁니다. */
+.goal {
+  @include card;
+  display: flex;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)